add link to fips-bundle-test library (#410)
jenkinsci/jep: jep/237/README.adoc

For these reasons it is not currently intended to add unit test level checks into the ecosystem.

> To facilitate Test of some specific FIPS parts, a library has been created to use with `jenkins-test-harness`.
> This library can be find here https://github.com/jenkinsci/lib-fips-bundle-test

Reviewer: ```suggestion
To facilitate testing of specific FIPS parts, a library [fips-bundle-test](https://github.com/jenkinsci/lib-fips-bundle-test) has been created to use with `jenkins-test-harness`.

```
